### ISSUE TITLE
V560TNE Power State after Power Fail ON does not work with Battery 

### ISSUE THREAD

**Issue by philipandag** [Dasharo/dasharo-issues#1021]
### Component

Dasharo firmware

### Device

NovaCustom V56 14th Gen

### Dasharo version

v0.9.1-rc3

### Dasharo Tools Suite version

--

### Test case ID

PSF003.001

### Brief summary

The device does not power on on its own after reconnecting the battery. Only an AC adapter powers it on

### How reproducible

100%

### How to reproduce

Set the `Power state after power failure` field to `Powered On`, save, restart device. Unplug the AC adapter and the battery while the device is running to simulate power loss. Plug the battery back in.

### Expected behavior

It should probably boot?

### Actual behavior

It does not boot when the battery is reconnected, but connecting an AC adapter does make the device boot on its own. 

### Additional context

Not sure if this is an issue, test specification is not specific about this.

**Comment by mkopec** [comment]
Yeah the feature is mislabeled. On laptops this feature enables power on / wake on AC adapter attach.

**Comment by philipandag** [comment]
Will we do anything about it? At very least we could say that it is mislabeled in the test specification and say how to test it on battery powered devices.

**Comment by wessel-novacustom** [comment]
> Only an AC adapter powers it on

- This is how the function is intended. It's just the description in the UEFI firmware settings and/or in the documentation that should be updated.

**Comment by mkopec** [comment]
Updated in PR https://github.com/Dasharo/docs/commit/9ec919b755a095f2fd9c2d5ed9a85bbe711b3397#diff-52b2a160a1bc5d8673fafd1055406b4b52458cd3b21e225f55403760738b5cca
